recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -15,6 +15,12 @@ SRC_URI_append_mx6sl = " file://weston.config"
 INI_UNCOMMENT_ASSIGNMENTS_append_imx = " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'x11 wayland', 'xwayland=true', '', d)} \
 "
+INI_UNCOMMENT_ASSIGNMENTS_append_mx6dl = " \
+    use-g2d=1 \
+"
+INI_UNCOMMENT_ASSIGNMENTS_append_mx6q = " \
+    use-g2d=1 \
+"
```

Comment:
They do support it but I wonder which we ought to use by default. @thochstein can you check internally?

Reply:
Prabhu says this is a good idea.

Reply:
That's good; thanks for the feedback.